lib/Sema/SemaExpr.cpp

```diff
@@ -12359,6 +12359,50 @@ ExprResult Sema::ActOnRangeBoundsExpr(SourceLocation BoundsKWLoc,
                                        RParenLoc);
 }
 
+ExprResult Sema::CreateRangeBoundsExpr(SourceLocation BoundsKWLoc,
+                                       Expr *LowerBound, Expr *UpperBound,
+                                       RelativeBoundsClause *Relative,
+                                       SourceLocation RParenLoc) {
+  RangeBoundsExpr *Range = nullptr;
+  ExprResult Result =
+      ActOnRangeBoundsExpr(BoundsKWLoc, LowerBound, UpperBound, RParenLoc);
+  Range = cast<RangeBoundsExpr>(Result.get());
```

Comment:
ExprResult can be invalid or carry a value with it that is valid.  You need to check if Result.isInvalid() and return ExprError() if it is.   You can't call Result.get on an invalid result.